src/oasis-nexus/api.ts

```diff
@@ -811,6 +815,7 @@ export const useGetConsensusProposals: typeof generated.useGetConsensusProposals
             proposals: data.proposals.map(proposal => {
               return {
                 ...proposal,
+                network,
```

Comment:
I would expect `layer: 'consensus'`. How does `isItemInScope` work otherwise?
and how do SearchResultItemCore types allow this :scream:

Reply:
I am adding `layer` in the search hooks, here: https://github.com/oasisprotocol/explorer/pull/1192/files#diff-b129d850541c740e998adaf540687f07f1f6b8012922be706f48efea08bc36a5R236

Reply:
Never mind, moved it here, for easier reading.